css-brochure-form/styles.css

```diff
@@ -0,0 +1,70 @@
+h2 {
+  color: orangered;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-size: large;
+  text-align: center;
+  letter-spacing: 1px;
+  font-weight: 700;
+}
+
+form {
+  display: flex;
+  justify-content: center;
+}
+
+fieldset {
+  border: 0;
+}
+
+label {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-weight: bold;
+  font-size: small;
+  padding: 0 0 4px 10px;
+  letter-spacing: 0.8px;
+}
+
+input {
+  margin: 2px 0 16px;
+  padding: 16px 0 12px 10px;
+  border-radius: 5px;
+  border: 1.5px solid #ccc;
+  width: 420px;
+}
+
+input::placeholder {
+  color: #808080;
+  font-weight: 111;
+  font-size: 1.3em;
+  font-family: Atlasgrotesk, sans-serif;
+}
+
+select {
+  padding: 10px 0 10px 2px;
+  margin: auto;
+  border-radius: 5px;
+  width: 420px;
+  font-family:
+    "Brandon Grotesque",
+    sans-serif;
+  font-size: 1.1em;
+  font-weight: 500;
+}
+
+select > option {
+  padding-left: 0;
+}
+
+button {
+  font-family: Atlasgrotesk, sans-serif;
+  font-weight: bold;
+  color: white;
+  background-color: orangered;
+  border: 2px;
+  border-radius: 15px;
+  height: 34px;
+  width: 100px;
+  margin: 10px 0;
+  padding: 1px 0 4px;
+  float: right;
```

Comment:
I would avoid using float here, text align right should work well. float is only for wrapping text around images.